### PR TITLE
Allantools

### PR DESCRIPTION
**Pull request by aewallin**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/allantools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  sha256: cd715068538ac1e457d88251fa166e103c536a861edae4458e970b9d4edf316b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
```
Comment (xhochy) [inline recipes/allantools/meta.yaml]: These should all go into `host`

File: recipes/allantools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - numpydoc
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - pytest
```
Comment (xhochy) [inline recipes/allantools/meta.yaml]: Upstream only seems to require numpy and scipy: https://github.com/aewallin/allantools/blob/32c9055a4811899fde6ae2013dbf40e8aedf7ab6/setup.py#L23

**Comment by aewallin** [comment]
I'm not sure how LICENSE could be included in the tar-ball built by setuptools and hosted on PyPi... any ideas?

**Comment by xhochy** [comment]
See https://github.com/TDAmeritrade/stumpy/pull/4 for an example

**Comment by aewallin** [comment]
FWIW I am willing to maintain the recipe to the best of my ability.

File: recipes/allantools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{% set name = "allantools" %}
+{% set version = "2019.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b9/63/3a998237dde66b844a5f3d0d90f62dcfde668ed5f0fc8257c7d57023f8b4/AllanTools-2019.9.tar.gz
```
Comment (chrisburr) [inline recipes/allantools/meta.yaml]: This should use the predictable URL, see the example for how to construct it

File: recipes/allantools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - numpydoc
+    - pytest-runner
+    - setuptools
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    # - matplotlib
```
Comment (chrisburr) [inline recipes/allantools/meta.yaml]: Remove these comments if they're not needed

File: recipes/allantools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    - allantools.dataset
+    - allantools.ci
+    - allantools.noise
+    - allantools.noise_kasdin
+    - allantools.plot
+    - allantools.realtime
+
+about:
+  home: https://github.com/aewallin/allantools
+  license: LGPLv3+
```
Comment (chrisburr) [inline recipes/allantools/meta.yaml]:
```suggestion
  license: LGPL-3.0-or-later
```

**Comment by aewallin** [comment]
I don't understand why the linux build fails now: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=74354&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=205&lineEnd=208&colStart=1&colEnd=39

**Comment by chrisburr** [comment]
It looks like a bug in the CI, I’ll take a look shortly

**Pull request merged**